src/index.js

```diff
@@ -15,8 +15,9 @@ const { koreanObserver } = require("./scripts/korean-channel");
 const { resourcesObserver } = require("./scripts/resource-channels");
 const { manualUnMute } = require("./scripts/users/permissions");
 const { regularQualifyCheck } = require("./scripts/users/user-utilities");
-const { unPin50thMsg, getAllChannels, ping } = require("./scripts/utilities");
-const { addBookmark, removeBookmark } = require("./scripts/bookmarks");
+const { isDm, handleDmReactionAdd } = require("./scripts/users/dm/dm");
+const { addBookmark, removeBookmark } = require("./scripts/users/dm/bookmarks");
+const { unPin50thMsg, getAllChannels, logMessageDate, ping } = require("./scripts/utilities");
```

Comment:
Not sure if you meant to re-add `logMessageDate` here. I don't believe we're using it anymore and @ well-balanced removed it with #32. To save you the hassle I'll just do a quick commit.